draft-ietf-quic-transport.md

```diff
@@ -851,6 +851,12 @@ limits are set in the transport parameters (see
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
 unidirectional and bidirectional streams.
 
+If a max_streams transport parameter or MAX_STREAMS frame is received with a
+value greater than 2^60, this allows a maximum stream ID that cannot be
+expressed as a variable-length integer (see {{integer-encoding}}).
```

Comment:
Maybe better: ... this value would be larger than the maximum stream ID that can be expresed as a variable-length integer.

Reply:
I think I'm ok with the existing text at this point, but if others prefer yours, I'll change it.

Reply:
If we're bike-shedding the grammar, I'd vote for `... this would allow a maximum stream ID ...`. Using "would" correlates better to the fact that this is indeed a protocol error and therefore isn't actually allowing anything.

Reply:
SG, done.